src/libs/runtime-utils.js

```diff
@@ -42,7 +42,10 @@ export const defaultLocation = 'US'
 export const defaultComputeZone = 'US-CENTRAL1-A'
 export const defaultComputeRegion = 'US-CENTRAL1'
 
-export const defaultAutoPauseThreshold = 30
+export const defaultAutopauseThreshold = 30
+export const defaultAutopauseDisabledValue = 0
+export const isAutopauseEnabled = threshold => threshold > defaultAutopauseDisabledValue
```

Comment:
A comment could be good here:
```suggestion
// Leonardo considers autopause disabled when the threshold is set to 0
export const isAutopauseEnabled = threshold => threshold > defaultAutopauseDisabledValue
```